src/middlewares/__tests__/isAuth.test.js

```diff
@@ -0,0 +1,141 @@
+const jwt = require('jsonwebtoken');
+const isAuth = require('../isAuth');
+
+const validToken = { tokenValue: 'validToken', expires: 'validTime' };
+const validTokenWithNoValidTime = {
+  tokenValue: 'invalidToken',
+  expires: 'validTime',
+};
+const invalidToken = {
+  tokenValue: 'invalidToken',
+  expires: 'validTime',
+};
```

Comment:
Both these tokens are the same. Properties of one need to be changed.
Also rename one to `validTokenWithInvalidTime` and other to `invalidTokenWithValidTime`.